fiware-region-sanity-tests/commons/constants.py

```diff
@@ -88,6 +93,3 @@
 
 # ENDPOINTS
 ENDPOINT_TYPE_PUBLIC_URL = "publicURL"
-
-# TYPES
-TEST_TEXT_CONTENT_TYPE = "text/plain"
```

Comment:
Finally, swift client accepts text_files and binary_files in the same way, so it is unnecessary to specify the content_type.